src/components/Snippet/FileManager.vue

```diff
@@ -0,0 +1,215 @@
+<script>
+import { setTimeout } from 'timers';
+
+import {
+  Upload as GirderUpload,
+  UpsertFolder as GirderUpsertFolder,
+  DataBrowser as GirderDataBrowser,
+  Breadcrumb as GirderBreadcrumb,
+} from '../';
+import {
+  getLocationType,
+  isRootLocation,
+  createLocationValidator,
+} from '../../utils';
+
+export default {
+  components: {
+    GirderBreadcrumb,
+    GirderUpload,
+    GirderUpsertFolder,
+    GirderDataBrowser,
+  },
+
+  props: {
+    location: {
+      type: Object,
+      validator: createLocationValidator(true),
```

Comment:
This component also needs the additional validator inside created()

Reply:
Good catch.